timeline-chart/src/time-graph-font-controller.ts

```diff
@@ -62,6 +63,9 @@ export class FontController {
 
         let fontName: string | undefined;
         if (size) {
+            if(size < this.minFonSize) {
+                size = this.minFonSize;
+            }
```

Comment:
nit: size = Math.max(size, this.minFonSize)

Reply:
Thank you.